carma_wm_ctrl/src/WMBroadcasterNode.cpp

```diff
@@ -68,8 +68,11 @@ int WMBroadcasterNode::run()
   georef_sub_ = cnh_.subscribe("georeference", 1, &WMBroadcaster::geoReferenceCallback, &wmb_);
   // Geofence Sub
   geofence_sub_ = cnh_.subscribe("geofence", 200, &WMBroadcaster::geofenceCallback, &wmb_);
+  // External Map Msg Sub
+  incoming_map_sub_ = cnh_.subscribe("incoming_map", 200, &WMBroadcaster::externalMapMsgCallback, &wmb_);
```

Comment:
queue size can be smaller, map messages publish at 1 Hz, since we would probably never be in range of more than ~5 intersections that's 5 hz max. I think a queue size of 10 or 20 would be sufficient.

Reply:
Reduced to 20